README.md

```diff
@@ -113,28 +113,28 @@ def filter_genre(initial_dataset: pd.DataFrame, selected_genre):
 
 *This is the execution graph of the scenario we are implementing*
 
-<div align="center"><img src="readme_img/readme_exec_g.png" alt="Taipy Core Graph"  width="60%"/></div>
+<div align="center"><img src="readme_img/readme_exec_graph.png" alt="Taipy BE Graph"  width="40%"/></div>
 
 
 ### Taipy Studio - The easy peasy way
 *You can use the Taipy Studio extension in VSCode to configure your sequence with no code*
```

Comment:
Maybe, we could put the link here to Taipy Studio. What do you think?

Reply:
I put it below after the gif on taipy studio